src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -594,8 +594,8 @@
       "defaultValue": "false"
     },
     "dspGenerator": {
-      "displayName": "Import DSP",
-      "description": "Import a Design System Package (DSP) to override Material colors",
+      "displayName": "Import Uno DSP",
+      "description": "Import a Uno DSP (Design System Package) theme file to override Material colors",
```

Comment:
@carldebilly, @rajamatt The primary goal was that we should not mention "Design System Package" anywhere anymore no?